cmd/podmanV2/pods/prune.go

```diff
@@ -0,0 +1,56 @@
+package pods
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/cmd/podmanV2/utils"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/spf13/cobra"
+)
+
+var (
+	pruneOptions = entities.PodPruneOptions{}
+)
+
+var (
+	pruneDescription = fmt.Sprintf(`podman pod prune Removes all exited pods`)
+
+	pruneCommand = &cobra.Command{
+		Use:     "prune [flags]",
+		Short:   "Remove all stopped pods and their containers",
+		Long:    pruneDescription,
+		RunE:    prune,
+		Example: `podman pod prune`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: pruneCommand,
+		Parent:  podCmd,
+	})
+	flags := pruneCommand.Flags()
+	flags.BoolVarP(&pruneOptions.Force, "force", "f", false, "Force removal of all running pods.  The default is false")
+}
+
+func prune(cmd *cobra.Command, args []string) error {
```

Comment:
We need to check the arguments here as v1 does:

```
~ $ podman pod prune 234908234908234 23423423
Error: `podman pod prune` takes no arguments
```

Reply:
fixed.